src/aquarium-optimized/dawn/imgui_impl_dawn.cpp

```diff
@@ -314,19 +347,16 @@ bool ImGui_ImplDawn_CreateDeviceObjects(int MSAASampleCount, bool enableAlphaBle
     rasterizationState.depthBiasClamp      = 0.0;
 
     // create graphics mPipeline
-    utils::ComboRenderPipelineDescriptor mPipelineDescriptor(mContextDawn->mDevice);
-    mPipelineDescriptor.layout                    = mPipelineLayout;
-    mPipelineDescriptor.vertexStage.module        = mVsModule;
-    mPipelineDescriptor.cFragmentStage.module     = mFsModule;
-    mPipelineDescriptor.cVertexState              = mVertexStateDescriptor;
-    mPipelineDescriptor.depthStencilState         = &mPipelineDescriptor.cDepthStencilState;
-    mPipelineDescriptor.cDepthStencilState.format = wgpu::TextureFormat::Depth24PlusStencil8;
-    mPipelineDescriptor.cColorStates[0]           = ColorStateDescriptor;
-    mPipelineDescriptor.cColorStates[0].format    = mFormat;
-    mPipelineDescriptor.cDepthStencilState.depthWriteEnabled = false;
-    mPipelineDescriptor.cDepthStencilState.depthCompare      = wgpu::CompareFunction::Always;
+    wgpu::RenderPipelineDescriptor mPipelineDescriptor;
+    mPipelineDescriptor.layout             = mPipelineLayout;
+    mPipelineDescriptor.vertexStage        = vertexStageDescriptor;
+    mPipelineDescriptor.fragmentStage      = &fragmentStageDescriptor;
+    mPipelineDescriptor.vertexState        = &mVertexStateDescriptor;
+    mPipelineDescriptor.depthStencilState  = &depthStencilStateDescriptor;
+    mPipelineDescriptor.colorStateCount    = 1;
+    mPipelineDescriptor.colorStates        = &ColorStateDescriptor;
     mPipelineDescriptor.primitiveTopology  = wgpu::PrimitiveTopology::TriangleList;
-    mPipelineDescriptor.sampleCount                          = MSAASampleCount;
+    mPipelineDescriptor.sampleCount        = MSAASampleCount;
```

Comment:
I remember we removed the criteria of alignment here. Can you do clang-format for all the changes?